src/selector-parse.ts

```diff
@@ -1,9 +1,12 @@
+import { BlockQueryProperties } from './abstracts';
+
 enum SelectorSyntax {
 	OPEN_BRACKET = '[',
 	CLOSED_BRACKET = ']',
 	EQUALS = '=',
 	DOUBLE_QUOTE = '\"',
-	SINGLE_QUOTE = '\''
+    SINGLE_QUOTE = '\'',
+    TYPE_SELECTOR = '.'
```

Comment:
Do we have a doc I contribute to on the selectors? I think a `:` colon character would better match opcode types.

Reply:
Or we could do `:type(motion)`. Like `:not(...)` https://developer.mozilla.org/en-US/docs/Web/CSS/:not

Reply:
@mzgoddard Definitely. Feel free to PR on the README!

Reply:
I'm feeling conservative on this and want to stick to the current documentation for now.